### PR TITLE
Add youtube-nocookie domain

### PR DESCRIPTION
**Pull request by finn0**
Add www.youtube-nocookie.com domain to Youtube's domain list.

### CODE REVIEW

File: kitsune/sumo/parser.py [hunk 1]
```diff
@@ -403,4 +403,4 @@ def _is_youtube_url(url):
     """Returns true if the URL is to youtube."""
     parsed_url = urlparse(url)
     netloc = parsed_url.netloc
-    return netloc in ["youtu.be", "youtube.com", "www.youtube.com"]
+    return netloc in ["youtu.be", "youtube.com", "www.youtube.com", "www.youtube-nocookie.com"]
```
Comment (akatsoulas) [inline kitsune/sumo/parser.py:406]: shall we also add another entry without the `www` subdomain?

Comment (finn0) [reply kitsune/sumo/parser.py:406]: I also thought about that, but when you open https://youtube-nocookie.com, you will get a certificate error message, saying that certificate isn't valid for this domain. Once I had reported this to the Google, but never get a reply. If you want, I can add domain without `www` subdomain.

Comment (akatsoulas) [reply kitsune/sumo/parser.py:406]: nice catch. I believe it's fine as is. We can revisit if needed

**Review by akatsoulas — APPROVED** [review]

File: kitsune/sumo/parser.py (hunk 1 above)
Comment (finn0) [reply kitsune/sumo/parser.py:406]: Youtube also uses these domains, https://youtubeeducation.com, https://youtubekids.com, and https://yt.be, but don't know whether these domains are useful for us or not. Should I add these domains also?

Comment (LeoMcA) [reply kitsune/sumo/parser.py:406]: I don't think we'll be embedding youtube videos from those domains (which is the ultimate purpose of this function) so I think we're ok with the urls we've now got there

**Pull request merged**